docs/src/pages/[platform]/theming/theme-provider/examples/CustomThemeExample.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  Card,
+  Heading,
+  Text,
+  ThemeProvider,
+  useTheme,
+} from '@aws-amplify/ui-react';
+
+export const CustomThemeExample = () => {
+  const { tokens } = useTheme();
```

Comment:
We can get rid of this line and importing the `useTheme` hook since it isn't being used in this example.